client/dashboard/profile-wizard/steps/start/index.js

```diff
@@ -269,15 +269,17 @@ class Start extends Component {
 					</Button>
 				</Card>
 
-				<p>
-					<Button
-						isLink
-						className="woocommerce-profile-wizard__skip"
-						onClick={ () => this.setState( { showUsageModal: true, continueAction: 'skip' } ) }
-					>
-						{ sprintf( __( 'Proceed without %s', 'woocommerce-admin' ), pluginNamesString ) }
-					</Button>
-				</p>
+				{ pluginNamesString && (
```

Comment:
Minor but would it be easier to read this if we checked to see if `pluginsToInstall` size was !== 0? Only reason I mention it is I had to go back and look at the code above to see how `pluginNamesString` was being defined.

Reply:
Agreed; that's a bit more readable.  Updated in https://github.com/woocommerce/woocommerce-admin/pull/3485/commits/d4aaf6c9fd6970a4647189ccadd47096e10b1b24